tools/profiling/timer.py

```diff
@@ -4,8 +4,8 @@
 
 from .constants import INTERVAL_SECONDS
 
-TIMER = signal.ITIMER_PROF
-TIMER_SIGNAL = signal.SIGPROF
+TIMER = signal.ITIMER_PROF  # type: ignore[attr-defined,unused-ignore]  # pylint: disable=no-member  # noqa: E501  # windows doesn't have ITIMER_PROF
```

Comment:
Put those under an os.name check then same as the one below and explain in the comment why and in the type ignore that it's just the linters not undestanding the os.name check